### ISSUE TITLE
NumberAxis3D.getTickLabelFormatter() never used?

### ISSUE THREAD

**Issue by ThomasDaheim** [jfree/orson-charts#4]
Hi,

I was trying to change to format of the tick label using NumberAxis3D.setTickLabelFormatter - with no results. No matter what I set here the format is always the same.

When checking, how the formatter is used in the library I get

```
Usages of NumberAxis3D.getTickLabelFormatter [no occurrences]                
```

so it seems that this feature isn't used at all?

**Comment by ThomasDaheim** [comment]
Have to answer myself with some findings :-)

As I can see from debugging, my value of tickLabelFormatter gets overwritten in Chart3D.draw -> Chart3D.findAxisTickUnits -> NumberAxis3D.selectTick with the value of this.tickSelector.getCurrentTickLabelFormat(). No wonder my formats don't get used...

Unfortunately, I don't see any method for NumberTickSelector.setCurrentTickLabelFormat :-( So how could I change the format of the output?

**Comment by ThomasDaheim** [comment]
See my pull request for this one

**Comment by jfree** [comment]
Hmm, looks like I applied your fix in the 2.0 release, should have updated this ticket.  Thanks for pointing out the bug!
